Inc_Heated_Cylinders/flow_cylinder.cfg

```diff
@@ -0,0 +1,261 @@
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%                                                                              %
+% SU2 configuration file                                                       %
+% Case description: Steady incompressible laminar flow around heated cylinders %
+%                                                                              %
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+% ------------- DIRECT, ADJOINT, AND LINEARIZED PROBLEM DEFINITION ------------%
+%
+% Physical governing equations (EULER, NAVIER_STOKES,
+%                               WAVE_EQUATION, HEAT_EQUATION, FEM_ELASTICITY,
+%                               POISSON_EQUATION)                           
+SOLVER= INC_NAVIER_STOKES
+%
+% If Navier-Stokes, kind of turbulent model (NONE, SA)
+KIND_TURB_MODEL= NONE
+%
+% Restart solution (NO, YES)
+RESTART_SOL= NO
+%
+% Objective function in gradient evaluation   (DRAG, LIFT, SIDEFORCE, MOMENT_X,
+%                                             MOMENT_Y, MOMENT_Z, EFFICIENCY,
+%                                             EQUIVALENT_AREA, NEARFIELD_PRESSURE,
+%                                             FORCE_X, FORCE_Y, FORCE_Z, THRUST,
+%                                             TORQUE, TOTAL_HEATFLUX,
+%                                             MAXIMUM_HEATFLUX, INVERSE_DESIGN_PRESSURE,
+%                                             INVERSE_DESIGN_HEATFLUX, SURFACE_TOTAL_PRESSURE,
+%                                             SURFACE_MASSFLOW, SURFACE_STATIC_PRESSURE, SURFACE_MACH)
+% For a weighted sum of objectives: separate by commas, add OBJECTIVE_WEIGHT and MARKER_MONITORING in matching order.
+OBJECTIVE_FUNCTION= TOTAL_HEATFLUX
+%
+% List of weighting values when using more than one OBJECTIVE_FUNCTION. Separate by commas and match with MARKER_MONITORING.
+OBJECTIVE_WEIGHT = 1.0
+%
+% Read binary restart files (YES, NO)
+READ_BINARY_RESTART = NO
+%
+% Write binary restart files (YES, NO)
+WRT_BINARY_RESTART = NO
```

Comment:
This option is deprecated, right? And handled via OUTPUT_FILES

Reply:
Yes, I'll change